src/ig/IGExporter.ts

```diff
@@ -251,7 +251,38 @@ export class IGExporter {
     const inputIndexMarkdownPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.md');
     const inputIndexXMLPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.xml');
     let generation: ImplementationGuideDefinitionPageGeneration = 'markdown';
-    if (existsSync(inputIndexMarkdownPageContentPath)) {
+    let filePath: string;
+    if (existsSync(inputIndexMarkdownPageContentPath)) filePath = inputIndexMarkdownPageContentPath;
+    if (existsSync(inputIndexXMLPageContentPath)) filePath = inputIndexXMLPageContentPath;
+    if (existsSync(inputIndexMarkdownPagesPath)) filePath = inputIndexMarkdownPagesPath;
+    if (existsSync(inputIndexXMLPagesPath)) filePath = inputIndexXMLPagesPath;
+
+    if (this.config.indexPageContent) {
+      ensureDirSync(pageContentExportPath);
+      logger.info('Generated index.md based on "indexPageContent" in config.yaml.');
+      const warning = warningBlock('<!-- index.md {% comment %}', '{% endcomment %} -->', [
+        'To change the contents of this file, edit the "indexPageContent" attribute in the tank config.yaml file',
+        'or provide your own index file in the ig-data/input/pagecontent or ig-data/input/pages folder.',
+        'See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root.input'
```

Comment:
Could this use `path.sep` instead of `/` to give some love to us Windows users?

Reply:
No love for the Windows users.

But also, yeah of course.